code/integration-tests/src/androidTest/java/com/adobe/marketing/mobile/integration/identity/IdentityIntegrationTests.kt

```diff
@@ -237,10 +239,44 @@ class IdentityIntegrationTests {
         assertFalse(countDownLatch.await(200, TimeUnit.MILLISECONDS))
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = TEST_TIMEOUT)
```

Comment:
I feel 500ms could cause random test failures in tests where we wait for multiple latches. Can you increase the default to 1000ms? It should not increase the total execution time as we only wait for TEST_TIMEOUT in worst case?